File: CodeSamples/python/swagger_client/models/check_transaction_type.py
# coding: utf-8

class CheckTransactionType(object):
    UNKNOWN = "Unknown"
    AUTHORIZE = "Authorize"
    VOID = "Void"
    OVERRIDE = "Override"
    PAYROLL = "Payroll"
    RECURRING = "Recurring"
    REFUND = "Refund"

    swagger_types = {
    }

    attribute_map = {
    }

    def __init__(self):  # noqa: E501
        """CheckTransactionType - a model defined in Swagger"""  # noqa: E501
        self.discriminator = None

File: CodeSamples/nodejs/src/model/CheckTransactionType.js
export default class CheckTransactionType {
  /**
   * value: "Unknown"
   * @const
   */
  Unknown = 'Unknown';

  /**
   * value: "Authorize"
   * @const
   */
  Authorize = 'Authorize';

  /**
   * value: "Void"
   * @const
   */
  Void = 'Void';

  /**
   * value: "Override"
   * @const
   */
  Override = 'Override';

  /**
   * value: "Payroll"
   * @const
   */
  Payroll = 'Payroll';

  /**
   * value: "Recurring"
   * @const
   */
  Recurring = 'Recurring';

  /**
   * value: "Refund"
   * @const
   */
  Refund = 'Refund';

  static constructFromObject (object) {
    return object;
  }
}
